cruft/_cli.py

```diff
@@ -231,3 +231,19 @@ def update(
         strict=strict,
     ):
         raise typer.Exit(1)
+
+
+@app.command(
+    short_help="Show the diff between the project and the current cruft template.",
+    help=_get_help_string(_commands.diff),
+)
+def diff(
+    project_dir: Path = typer.Option(
+        Path("."), "--project-dir", "-p", help="Path to the project directory.", show_default=False
+    ),
+    exit_code: bool = typer.Option(
```

Comment:
Let's call this the check flag. it's more in line with other liners

Reply:
More in line with other linters, less in line with git. But who uses git anyways…

Reply:
One argument to keep it as `--exit-code`: `--check` natural shortcode is `-c`, but that would conflict with `--checkout`. If you want me to change it, give me the shortcode you want for `--check`.

Reply:
Thanks for the explanation. That makes sense. 

PS - I slightly edited your previous comment about git to be more in line with our [CoC](https://cruft.github.io/cruft/docs/contributing/3.-code-of-conduct/). I would again like to apologize if my review comments  made you feel like I was being inconsiderate, that was not my intention, but let's please keep the discussion polite. We are all trying our best to contribute to a community project here :)

Reply:
Sorry for the sarcastic comment. Didn't know it wasn't in line with the CoC.